src/military/military.cpp

```diff
@@ -239,10 +239,12 @@ auto province_is_under_siege(sys::state const& state, T ids) {
 
 template<typename T>
 auto battle_is_ongoing_in_province(sys::state const& state, T ids) {
-	ve::apply([&](dcon::province_id p) {
-		auto battles = state.world.province_get_land_battle_location(p);
+	ve::apply(
+			[&](dcon::province_id p) {
+				auto battles = state.world.province_get_land_battle_location(p);
 				return battles.begin() != battles.end();
-	}, ids);
+			},
+			ids);
```

Comment:
totally honesty here: I'm getting real tired of fighting the formatter at some places